tests/storage/test_wffc.py

```
@@ -295,32 +333,43 @@ def test_wffc_add_dv_to_vm_with_data_volume_template(
+        pytest.mark.polarion("CNV-4743"),
+    ],
+    indirect=True,
+)
 def test_wffc_vm_with_two_data_volume_templates(
+    request,
     unprivileged_client,
     namespace,
     storage_class_matrix_wffc_matrix__module__,
+    rhel10_data_source_scope_module,
 ):
     storage_class = [*storage_class_matrix_wffc_matrix__module__][0]
```

Ruff (0.15.9)

[warning] Prefer `next(iter(storage_class_matrix_wffc_matrix__module__))` over single element slice

Replace with `next(iter(storage_class_matrix_wffc_matrix__module__))`

(RUF015)